app/static/src/app/components/options/ParameterValues.vue

```diff
@@ -71,11 +73,13 @@ export default defineComponent({
         const timestampParamNames = () => paramNames.value.map((name: string) => name + Date.now());
 
         const paramKeys = ref(timestampParamNames());
-        const odinSolution = computed(() => store.state.run.result?.solution);
+        const odinSolution = computed(() => store.state.run.resultOde?.solution);
 
         const updateValue = (newValue: number, paramName: string) => {
             store.commit(`run/${RunMutation.UpdateParameterValues}`, { [paramName]: newValue });
-            store.commit(`modelFit/${ModelFitMutation.SetFitUpdateRequired}`, { parameterValueChanged: true });
+            if (isFit.value) {
```

Comment:
Comment has nothing to do with the PR but I was wondering if we can rethink the approach, perhaps design a metadata driven approach in choosing appType.

Reply:
Do you mean that there should be more metadata-driven flexibility in what an app does, as opposed to belonging to one of three fixed app types, with behaviour defined in the code conditional on these? We did discuss that a bit early on as an option - but it seemed that the app type was quite fundamental and an acceptable assumption which would reduce complexity overall.

Reply:
This might be good to talk about in a retrospective of the app once we get this first set of app types in, and start thinking about future ones - how will we keep things extensible and reusable in future. the overall design is sufficiently different from hint that I suspect intuition from there might be misleading but some sort of "traits" based idea (i.e., my app needs this sort of thing from these components) might be more extensible than the current orientation where the component looks at the type of app and then provides features

Reply:
Yeah, so rather than having a single app type, an app would have one or more of these trait properties. That's more flexible, though could lead to ambiguity in how traits can be combined..